packages/presentation/lib/const.dart

```diff
@@ -80,18 +93,72 @@ class HomeScreenStyles {
   );
 }
 
+class MovieDetailsScreenSizes {
+  static const double ratingStarSize = 24;
+  static const double navigationIconSize = 21;
+}
+
+class MovieDetailsScreenStyles {
+  static const movieNameStyle = TextStyle(
+    fontSize: 24,
+    fontWeight: FontWeight.w600,
+    fontFamily: AppFonts.sfProTextFontName,
+    color: AppColors.white,
+  );
+
+  static const movieAdditionalDataStyle = TextStyle(
+    fontSize: 16,
+    fontFamily: AppFonts.sfProTextFontName,
+    color: AppColors.transparentWhite,
+  );
+
+  static const movieDescriptionHeaderStyle = TextStyle(
+    fontSize: 18,
+    fontFamily: AppFonts.sfProTextFontName,
+    color: AppColors.white,
+  );
```

Comment:
let's not spawn a lot of almost identical styles
e.g we can choose some general parameter for them(color) and create one style
static TextStyle white(
    double fontSize, {
    FontWeight fontWeight = AppFonts.regular,
    TextDecoration decoration = TextDecoration.none,
    double? lineHeight,
    double? opacity,
    FontStyle fontStyle = FontStyle.normal,
    double? letterSpacing,
  }) {
    return TextStyle(
      letterSpacing: letterSpacing,
      fontStyle: fontStyle,
      color: AppColors.white.withOpacity(opacity ?? 1.0),
      fontSize: fontSize,
      fontWeight: fontWeight,
      decoration: decoration,
      height: (lineHeight ?? fontSize) / fontSize,
    );
  }